src/plugins/audio-stream/menu.ts

```
@@ -0,0 +1,127 @@
+import prompt from 'custom-electron-prompt';
+
+import { t } from '@/i18n';
+import promptOptions from '@/providers/prompt-options';
+
+import type { MenuContext } from '@/types/contexts';
+import type { MenuTemplate } from '@/menu';
+
+import {
+  type AudioStreamConfig,
+  defaultAudioStreamConfig,
+} from './config';
```

ESLint

[error] `./config` import should occur before type import of `@/types/contexts`

(importPlugin/order)